### PR TITLE
vote: update(docs): GOVERNANCE.md

### PR DESCRIPTION
**Pull request by incertum**
Signed-off-by: Melissa Kilby <melissa.kilby.oss@gmail.com>
Co-authored-by: Leonardo Grasso <me@leonardograsso.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

/kind documentation

> /kind feature

> /kind incubating

> /kind officialsupport

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area governance

> /area proposals

> /area utils

**What this PR does / why we need it**:

* Add additional requirements to the road to maintainership.
* Add a clause for clarification of measurement of these criteria.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: GOVERNANCE.md
Comment (leogr) [inline GOVERNANCE.md:135]:
```suggestion
 - Domain expertise and a good understanding of the code-base of those areas.
```

It seems to be redundant with line 140.

**Review by leogr — COMMENTED** [review]
Thank you for this PR :heart: 

I've just left a few comments. Otherwise SGTM.

I'm just unsure about one point: I guess it makes sense to have an ingress barrier like this for Core Maintainers but perhaps not for non-Core maintainers. May we want to move those criteria to the Core Maintainers section? :thinking: 

Also, to change the GOVERNACE.md we need a supermajority vote, so I'm putting this on
/hold
just to avoid the PR gets merged by mistake before the votation. Once we have the definitive version of the doc we can start the votation and unhold the PR.
Alternatively, if we believe those are just guidelines and not hard-requirements, we can move them into MAINTAINERS_GUIDELINES.md which does not require a super-majority vote.

**Comment by incertum** [comment]
Thank you @leogr! Worked in the feedback. +1 on distinguishing between maintainers and core-maintainers criteria.

**Comment by maxgio92** [comment]
Thanks @incertum! I agree with you all about to distinguish the requirements between core and non-core maintainers

**Comment by leogr** [comment]
Dear @falcosecurity/core-maintainers :mega: 

Any change to our [GOVERNANCE.md](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md) requires a [supermajority vote](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#supermajority-vote) of [Core Maintainers](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#core-maintainers).

So with this comment, I officially open the vote for the changes proposed by @incertum :partying_face: 

/unhold

The voting **closes on 2022-11-24** or when we reach two thirds of votes in favor, whichever comes first.
Currently, we have [12 eligible voters](https://github.com/falcosecurity/evolution/blob/main/MAINTAINERS.md#core-maintainers), so the proposal is considered **successful with 8 votes in favor**.

Thanks again to @incertum for this proposal that aims to make our Governance document more precise :pray:

**Comment by leogr** [comment]
And ofc, **+1** for me

**Comment by Andreagit97** [comment]
+1 for me :fire:

**Comment by Issif** [comment]
+1 for me

**Comment by maxgio92** [comment]
+1 for me too

**Comment by Molter73** [comment]
+1 on my side

**Comment by zuc** [comment]
+1 from me as well!

File: GOVERNANCE.md
Comment (leogr) [reply GOVERNANCE.md:138]: It should be an idiom (_show someone the ropes_), not a typo.

**Comment by jasondellaluce** [comment]
+1

**Comment by cpanato** [comment]
+1

**Comment by LucaGuerra** [comment]
+1

**Comment by leogr** [comment]
> Dear @falcosecurity/core-maintainers mega
> 
> Any change to our [GOVERNANCE.md](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md) requires a [supermajority vote](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#supermajority-vote) of [Core Maintainers](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#core-maintainers).
> 
> So with this comment, I officially open the vote for the changes proposed by @incertum partying_face
> 
> The voting **closes on 2022-11-24** or when we reach two thirds of votes in favor, whichever comes first. Currently, we have [12 eligible voters](https://github.com/falcosecurity/evolution/blob/main/MAINTAINERS.md#core-maintainers), so the proposal is considered **successful with 8 votes in favor**.
> 
> The PR will not merged until then /hold
> 
> Thanks again to @incertum for this proposal that aims to make our Governance document more precise pray

To date, we have already obtained 10 votes, so we can officially state that this change is approved :rocket: 

Thank you all :hugs: 

/approve

**Review by leogr — APPROVED** [review]

**Pull request merged**
